x/tss/abci.go

```diff
@@ -117,11 +117,8 @@ func emitSignStartEvent(ctx sdk.Context, k types.TSSKeeper, voter types.InitPoll
 		}
 
 		// metrics for sign participation
-		telemetry.SetGaugeWithLabels(
-			[]string{types.ModuleName, "sign", "participation"},
-			float32(validator.ShareCount),
+		metrics.SetGaugeWithLabels([]string{types.ModuleName, "sign", "participation"}, float32(validator.ShareCount),
 			[]metrics.Label{
-				telemetry.NewLabel("timestamp", strconv.FormatInt(ctx.BlockTime().Unix(), 10)),
```

Comment:
what's the difference after remove the timestamp label?

Reply:
yeah, you were right, it does not solve anything because the keygen/sig IDs will always be different, so it will blow up metrics all the same